<launch>

    <node pkg="irols" name="arm_action" type="arm_action.py" output="screen"/>
    <node pkg="irols" name="land_action" type="land_action.py" output="screen"/>
    <node pkg="irols" name="wp_action" type="wp_action.py" output="screen"/>
    <node pkg="irols" name="seek_action" type="seek_action.py" output="screen"/>
    <node pkg="irols" name="flc_action" type="flc_action.py" output="screen"/>
    <node pkg="irols" name="fitness_action" type="fitness_action.py" output="screen"/>
    <node pkg="irols" name="land_mach_action" type="approach_action.py" output="screen"/>
    <node pkg="irols" name="ga_action" type="ga_action.py" output="screen"/>

    <rosparam command="load" file="$(find irols)/configs/x_fis.yaml" ns="x_fis"/>
    <rosparam command="load" file="$(find irols)/configs/y_fis.yaml" ns="y_fis"/>
    <rosparam command="load" file="$(find irols)/configs/z_fis.yaml" ns="z_fis"/>
    <rosparam command="load" file="$(find irols)/configs/T_fis.yaml" ns="T_fis"/>
</launch>
